server/src/main/java/com/genymobile/scrcpy/Server.java

```diff
@@ -2,7 +2,7 @@
 
 import android.os.BatteryManager;
 import android.os.Build;
-
+import android.system.Os;
```

Comment:
No need for this change, revert

Reply:
Fixed, thank you. For some reason I thought it was needed.